### PR TITLE
💬 Grammar fixes

### PR DESCRIPTION
**Pull request by pboling**
Minor fixes to grammar, adding a few missing words.

Screen Shot 2022-05-05 at 8 45 19 AM

### CODE REVIEW

**Comment by simi** [comment]
@bettymakes can you check please?

**Review by bettymakes — COMMENTED** [review]
👍 Both changes look good!

There's opportunity to refine the `signin_to_create_html` copy a bit more:

Please sign in to create **an ownership request** _(Suggestion)_
vs. 
Please sign in to create **a request for ownership** _(Currently in PR)_

My suggestion would be aligned with the copy as referenced under the `create_req` locale key. But to re-iterate, the current suggestion is grammatically acceptable as well 👍 .

File: config/locales/en.yml [hunk 1]
```diff
@@ -570,4 +570,4 @@ en:
     created_at: Created at
     no_ownership_requests: Requests to join your project will show up here. No ownership requests for %{gem} yet.
     create_req: Create ownership request
-    signin_to_create_html: Please <a href="https://rubygems.org/sign_in">sign in</a> to create request ownership.
+    signin_to_create_html: Please <a href="https://rubygems.org/sign_in">sign in</a> to create a request for ownership.
```
Comment (pboling) [inline config/locales/en.yml:573]:
```suggestion
    signin_to_create_html: Please <a href="https://rubygems.org/sign_in">sign in</a> to create an ownership request.
```

**Comment by pboling** [comment]
@bettymakes @simi I've made the suggested change.  It is an improvement.

**Comment by simi** [comment]
Thanks @pboling!

**Pull request merged**